Library/OcCryptoLib/Sha2-avx.h

```diff
@@ -0,0 +1,92 @@
+/** @file
+  Copyright (C) 2021, ISP RAS. All rights reserved.
+
+  All rights reserved.
+
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+**/
+
+#ifndef SHA2_AVX_H
```

Comment:
I think that header is now common between AVX and non-AVX version, so better to rename to `Sha2Internal.h` or something like that.